realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -226,9 +228,8 @@ protected void removeHandler() {
      * the last transaction was committed.
      *
      * @param destination file to save the Realm to.
-     * @throws java.io.IOException if any write operation fails.
      */
-    public void writeCopyTo(File destination) throws java.io.IOException {
+    public void writeCopyTo(File destination) {
```

Comment:
Also technically it is a breaking change. I would think that in this case actually exposing  it as an IO Exception is probably the  most accurate?